### PR TITLE
Fixtures community app

### PR DESCRIPTION
**Pull request by caioingber**
- Related Issue (include '#'): #523

- Description of changes made: Added fixtures for community app, updated serializers and added title return 

- Is the feature complete/bug resolved/etc..: For now, will add more depth for foreign keys and many to many relationships once models are up to date.

- Any known bugs/strange behavior: Not sure how to serialize the topics so that their titles are viewable, utilizing a `CharField()` didn't seem to work since they are in a list. 

- Is there specific feedback you would like on these changes: above

- Screenshot(s):

### CODE REVIEW

File: api/community/serializers.py [hunk 1]
```diff
@@ -7,12 +7,15 @@ class Meta:
...
+    group = serializers.CharField()
+    location = serializers.CharField()
+
     class Meta:
         model = Event
         fields = '__all__'
 
 class TopicSerializer(serializers.ModelSerializer):
-    class Meta:
+    class Mœeta:
```
Comment (ZakkMan) [inline api/community/serializers.py:18]: wow what's this?!

File: api/community/serializers.py [hunk 2]
```diff
@@ -22,6 +25,9 @@ class Meta:
         fields = '__all__'
 
 class MediaSerializer(serializers.ModelSerializer):
+    hashtag = serializers.CharField()
+    group = serializers.CharField()
+    topics = serializers.HyperlinkRelatedField(many=True, )
```
Comment (ZakkMan) [inline api/community/serializers.py:30]: This should be [HyperlinkedRelatedField](https://www.django-rest-framework.org/api-guide/relations/#hyperlinkedrelatedfield)

Comment (ZakkMan) [reply api/community/serializers.py:30]: This is throwing errors so until this is fixed I can't test out the fixtures

File: api/community/serializers.py (hunk 1 above)
Comment (caioingber) [reply api/community/serializers.py:18]: lol

File: api/community/serializers.py (hunk 2 above)
Comment (caioingber) [reply api/community/serializers.py]: removed serializer definition for topics for the time being

File: api/community/serializers.py (hunk 1 above)
Comment (Underwaterr) [reply api/community/serializers.py]: 👽

**Pull request merged**